src/hooks/useUniversalRouter.ts

```diff
@@ -38,19 +38,13 @@ export function useUniversalRouterSwapCallback(
         inputTokenPermit: options.permit,
         fee: options.feeOptions,
       })
-      const tx =
-        value && !isZero(value)
-          ? {
-              from: account,
-              to: UNIVERSAL_ROUTER_ADDRESS(chainId),
-              data,
-              value,
-            }
-          : {
-              from: account,
-              to: UNIVERSAL_ROUTER_ADDRESS(chainId),
-              data,
-            }
+      const tx = {
+        from: account,
+        to: UNIVERSAL_ROUTER_ADDRESS(chainId),
+        data,
+        // TODO: universal-router-sdk returns a non-hexlified value.
+        ...(value && !isZero(value) ? { value: toHex(value) } : {}),
```

Comment:
```ts
value: value && !isZero(value) ? toHex(value) : undefined
```

Reply:
I don't think we want the property value at all. Using this syntax would have the property defined as undefined, instead of not defined at all.

Reply:
it's the same thing afaik?

Reply:
Not the same, because you'll have 'value' in tx be true (ie Object.getOwnPropertyDescriptor).

Reply:
agreed not to do this for now if its diff